src/command_modules/azure-cli-network/azure/cli/command_modules/network/_params.py

```diff
@@ -194,15 +200,20 @@ def completer(prefix, action, parsed_args, **kwargs): # pylint: disable=unused-a
 register_cli_argument('network application-gateway url-path-map rule', 'http_settings', help='The name or ID of the HTTP settings. If not specified, the default for the map will be used.', completer=get_ag_subresource_completion_list('backend_http_settings_collection'))
 
 # Express Route Circuit Auth
-register_cli_argument('network express-route circuit-auth', 'authorization_name', name_arg_type)
-register_cli_argument('network express-route circuit-peering', 'peering_name', name_arg_type)
-register_cli_argument('network express-route circuit', 'circuit_name', name_arg_type, completer=get_resource_name_completion_list('Microsoft.Network/expressRouteCircuits'))
+register_cli_argument('network express-route circuit-auth', 'authorization_name', name_arg_type, id_part='child_name', help='Authorization name')
+register_cli_argument('network express-route circuit-auth', 'circuit_name', completer=get_resource_name_completion_list('Microsoft.Network/expressRouteCircuits'), id_part='name', help='Circuit name.')
+register_cli_argument('network express-route circuit-auth', 'authorization_key', help='Authorization key.')
 
 # Express Route Circuit Peering
 register_cli_argument('network express-route circuit-peering', 'peering_name', name_arg_type, id_part='child_name')
+register_cli_argument('network express-route circuit-peering', 'peering_type', choices=get_enum_choices(peeringType))
+register_cli_argument('network express-route circuit-peering', 'sku_family', choices=get_enum_choices(sku_family))
```

Comment:
>, choices=get_enum_choices(sku_family) [](start = 75, length = 38)

Are these enums modeled as strings in Swagger? If so, per Johan's guidance, we should use an enum completer instead of argparse choices, and put example values in the help text.